Error, while building using `go get`

Hello, i got error when trying to make promxy binary using `go get github.com/jacksontj/promxy/cmd/promxy@v0.0.75`

```
#13 [build 2/2] RUN go get github.com/jacksontj/promxy/cmd/promxy@v0.0.75
#13 sha256:66e7a228ee13f39af351f830ab8287d7a79263805b709ad6eabedd8eb3a960b4
#13 0.525 go: downloading github.com/jacksontj/promxy v0.0.75
#13 27.11 go: downloading github.com/go-kit/kit v0.10.0
#13 27.57 go: downloading github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
#13 27.60 go: downloading github.com/jessevdk/go-flags v1.5.0
#13 27.70 go: downloading github.com/julienschmidt/httprouter v1.3.0
#13 27.74 go: downloading github.com/prometheus/client_golang v1.11.1-0.20210607165600-196536534fbb
#13 27.91 go: downloading github.com/prometheus/common v0.30.0
#13 28.07 go: downloading github.com/prometheus/prometheus v1.8.2-0.20210707132820-dc8f50559534
#13 29.06 go: downloading github.com/sirupsen/logrus v1.7.0
#13 29.15 go: downloading go.uber.org/atomic v1.9.0
#13 29.23 go: downloading k8s.io/klog v1.0.0
#13 29.42 go: downloading github.com/go-logfmt/logfmt v0.5.0
#13 29.52 go: downloading github.com/prometheus/exporter-toolkit v0.6.1
#13 29.60 go: downloading gopkg.in/yaml.v2 v2.4.0
#13 29.68 go: downloading github.com/pkg/errors v0.9.1
#13 29.73 go: downloading golang.org/x/sys v0.0.0-20210615035016-665e8c7367d1
#13 30.97 go: downloading github.com/beorn7/perks v1.0.1
#13 31.00 go: downloading github.com/cespare/xxhash v1.1.0
#13 31.04 go: downloading github.com/cespare/xxhash/v2 v2.1.1
#13 31.08 go: downloading github.com/golang/protobuf v1.5.2
#13 31.24 go: downloading github.com/prometheus/client_model v0.2.0
#13 31.27 go: downloading github.com/prometheus/procfs v0.6.0
#13 31.51 go: downloading github.com/go-kit/log v0.1.0
#13 31.59 go: downloading github.com/alecthomas/units v0.0.0-20210208195552-ff826a37aa15
#13 31.62 go: downloading github.com/go-openapi/strfmt v0.20.1
#13 31.69 go: downloading github.com/prometheus/alertmanager v0.22.2
#13 32.52 go: downloading github.com/edsrzf/mmap-go v1.0.0
#13 32.56 go: downloading github.com/opentracing/opentracing-go v1.2.0
#13 32.64 go: downloading github.com/uber/jaeger-client-go v2.29.1+incompatible
#13 32.95 go: downloading github.com/gogo/protobuf v1.3.2
#13 34.65 go: downloading github.com/mwitkow/go-conntrack v0.0.0-20190716064945-2f068394615f
#13 34.69 go: downloading github.com/opentracing-contrib/go-stdlib v1.0.0
#13 34.73 go: downloading golang.org/x/net v0.0.0-20210610132358-84b48f89b13b
#13 35.76 go: downloading golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
#13 36.38 go: downloading github.com/golang/snappy v0.0.4
#13 36.45 go: downloading golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
#13 36.48 go: downloading github.com/json-iterator/go v1.1.11
#13 36.68 go: downloading github.com/oklog/ulid v1.3.1
#13 36.72 go: downloading golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
#13 36.79 go: downloading google.golang.org/protobuf v1.26.0
#13 37.77 go: downloading github.com/matttproud/golang_protobuf_extensions v1.0.1
#13 37.87 go: downloading golang.org/x/oauth2 v0.0.0-20210514164344-f6687ab2804c
#13 37.99 go: downloading github.com/aws/aws-sdk-go v1.38.60
#13 43.24 go: downloading github.com/Azure/azure-sdk-for-go v55.2.0+incompatible
#13 55.79 go: downloading github.com/Azure/go-autorest v14.2.0+incompatible
#13 56.17 go: downloading github.com/Azure/go-autorest/autorest v0.11.20
#13 56.29 go: downloading github.com/Azure/go-autorest/autorest/adal v0.9.14
#13 56.34 go: downloading github.com/hashicorp/consul/api v1.8.1
#13 56.46 go: downloading github.com/digitalocean/godo v1.62.0
#13 56.61 go: downloading github.com/miekg/dns v1.1.42
#13 56.86 go: downloading gopkg.in/fsnotify/fsnotify.v1 v1.4.7
#13 56.91 go: downloading google.golang.org/api v0.48.0
#13 61.96 go: downloading github.com/hetznercloud/hcloud-go v1.26.2
#13 62.32 go: downloading k8s.io/api v0.21.1
#13 63.35 go: downloading k8s.io/apimachinery v0.21.1
#13 63.64 go: downloading k8s.io/client-go v0.21.1
#13 64.46 go: downloading github.com/linode/linodego v0.28.5
#13 64.60 go: downloading github.com/docker/docker v20.10.7+incompatible
#13 65.61 go: downloading github.com/gophercloud/gophercloud v0.18.0
#13 66.38 go: downloading github.com/scaleway/scaleway-sdk-go v1.0.0-beta.7.0.20210223165440-c65ae3540d44
#13 66.61 go: downloading github.com/go-zookeeper/zk v1.0.2
#13 66.67 go: downloading github.com/asaskevich/govalidator v0.0.0-20200907205600-7a23bdc65eef
#13 66.74 go: downloading github.com/go-openapi/errors v0.20.0
#13 66.78 go: downloading github.com/mitchellh/mapstructure v1.4.1
#13 66.82 go: downloading go.mongodb.org/mongo-driver v1.5.1
#13 67.75 go: downloading github.com/go-openapi/swag v0.19.15
#13 67.83 go: downloading github.com/go-openapi/validate v0.20.2
#13 68.11 go: downloading github.com/stretchr/testify v1.7.0
#13 68.21 go: downloading go.uber.org/goleak v1.1.10
#13 68.29 go: downloading github.com/uber/jaeger-lib v2.4.1+incompatible
#13 68.38 go: downloading gopkg.in/yaml.v3 v3.0.0-20210107192922-496545a6307b
#13 68.47 go: downloading github.com/jpillora/backoff v1.0.0
#13 68.51 go: downloading github.com/shurcooL/httpfs v0.0.0-20190707220628-8d4bc4ba7749
#13 68.56 go: downloading github.com/shurcooL/vfsgen v0.0.0-20200824052919-0d455de96546
#13 68.63 go: downloading github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd
#13 68.66 go: downloading github.com/modern-go/reflect2 v1.0.1
#13 68.87 go: downloading github.com/Azure/go-autorest/autorest/date v0.3.0
#13 68.90 go: downloading github.com/Azure/go-autorest/autorest/to v0.4.0
#13 68.92 go: downloading github.com/Azure/go-autorest/autorest/validation v0.3.1
#13 68.95 go: downloading github.com/Azure/go-autorest/tracing v0.6.0
#13 68.98 go: downloading github.com/Azure/go-autorest/logger v0.2.1
#13 69.01 go: downloading github.com/form3tech-oss/jwt-go v3.2.2+incompatible
#13 69.09 go: downloading github.com/hashicorp/go-cleanhttp v0.5.1
#13 69.12 go: downloading github.com/hashicorp/go-hclog v0.12.2
#13 69.17 go: downloading github.com/hashicorp/go-rootcerts v1.0.2
#13 69.21 go: downloading github.com/hashicorp/serf v0.9.5
#13 69.57 go: downloading github.com/google/go-querystring v1.0.0
#13 69.65 go: downloading cloud.google.com/go v0.83.0
#13 71.37 go: downloading google.golang.org/appengine v1.6.7
#13 71.69 go: downloading google.golang.org/grpc v1.38.0
#13 72.13 go: downloading github.com/google/gofuzz v1.1.0
#13 72.17 go: downloading github.com/google/go-cmp v0.5.6
#13 72.32 go: downloading k8s.io/klog/v2 v2.9.0
#13 72.39 go: downloading sigs.k8s.io/structured-merge-diff/v4 v4.1.0
#13 72.71 go: downloading k8s.io/utils v0.0.0-20201110183641-67b214c5f920
#13 72.94 go: downloading github.com/imdario/mergo v0.3.5
#13 73.00 go: downloading github.com/spf13/pflag v1.0.5
#13 73.15 go: downloading golang.org/x/term v0.0.0-20210220032956-6a3ed077a48d
#13 73.21 go: downloading github.com/go-resty/resty/v2 v2.1.1-0.20191201195748-d7b97669fe48
#13 73.52 go: downloading github.com/docker/go-connections v0.4.0
#13 73.64 go: downloading github.com/docker/go-units v0.4.0
#13 73.67 go: downloading github.com/opencontainers/image-spec v1.0.1
#13 73.79 go: downloading github.com/containerd/containerd v1.4.3
#13 74.79 go: downloading github.com/docker/distribution v2.7.1+incompatible
#13 75.27 go: downloading github.com/opencontainers/go-digest v1.0.0
#13 75.33 go: downloading github.com/mailru/easyjson v0.7.6
#13 75.43 go: downloading github.com/go-openapi/analysis v0.20.0
#13 75.69 go: downloading github.com/go-openapi/jsonpointer v0.19.5
#13 75.80 go: downloading github.com/go-openapi/loads v0.20.2
#13 76.01 go: downloading github.com/go-openapi/runtime v0.19.28
#13 76.19 go: downloading github.com/go-openapi/spec v0.20.3
#13 76.53 go: downloading golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
#13 76.61 go: downloading golang.org/x/text v0.3.6
#13 78.36 go: downloading github.com/jmespath/go-jmespath v0.4.0
#13 78.81 go: downloading github.com/fatih/color v1.9.0
#13 78.84 go: downloading github.com/mattn/go-colorable v0.1.6
#13 78.89 go: downloading github.com/mattn/go-isatty v0.0.12
#13 78.92 go: downloading github.com/mitchellh/go-homedir v1.1.0
#13 78.94 go: downloading github.com/armon/go-metrics v0.3.3
#13 79.06 go: downloading github.com/googleapis/gax-go/v2 v2.0.5
#13 79.09 go: downloading go.opencensus.io v0.23.0
#13 79.40 go: downloading gopkg.in/inf.v0 v0.9.1
#13 79.44 go: downloading github.com/go-logr/logr v0.4.0
#13 79.47 go: downloading github.com/googleapis/gnostic v0.4.1
#13 79.85 go: downloading github.com/davecgh/go-spew v1.1.1
#13 79.91 go: downloading github.com/hashicorp/golang-lru v0.5.4
#13 79.95 go: downloading github.com/Microsoft/go-winio v0.4.16
#13 80.07 go: downloading github.com/go-stack/stack v1.8.0
#13 80.10 go: downloading github.com/josharian/intern v1.0.0
#13 80.13 go: downloading github.com/go-openapi/jsonreference v0.19.5
#13 80.16 go: downloading github.com/pmezard/go-difflib v1.0.0
#13 80.19 go: downloading github.com/hashicorp/go-immutable-radix v1.2.0
#13 80.24 go: downloading google.golang.org/genproto v0.0.0-20210604141403-392c879c8b08
#13 83.27 go: downloading sigs.k8s.io/yaml v1.2.0
#13 83.37 go: downloading github.com/PuerkitoBio/purell v1.1.1
#13 83.42 go: downloading golang.org/x/tools v0.1.3
#13 85.59 go: downloading github.com/golang/groupcache v0.0.0-20200121045136-8c9f03a8e57e
#13 85.65 go: downloading github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578
#13 86.55 go: downloading github.com/Azure/azure-sdk-for-go v0.2.0-beta
#13 87.66 go: downloading github.com/docker/docker v1.13.1
#13 90.93 go: downloading github.com/form3tech-oss/jwt-go v1.0.2
#13 93.86 go: downloading github.com/prometheus/prometheus v1.8.2
#13 94.73 go: downloading github.com/uber/jaeger-client-go v1.6.0
#13 94.92 go: downloading github.com/uber/jaeger-lib v1.5.0
#13 95.96 go: downloading k8s.io/client-go v1.5.2
#13 161.7 # github.com/jacksontj/promxy/pkg/promclient
#13 161.7 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/promclient/label.go:106:12: assignment mismatch: 2 variables but parser.Walk returns 1 values
#13 161.7 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/promclient/label.go:106:26: too many arguments in call to parser.Walk
#13 161.7 	have (context.Context, *LabelFilterVisitor, *parser.EvalStmt, parser.Expr, nil, nil)
#13 161.7 	want (parser.Visitor, parser.Node, []parser.Node)
#13 161.7 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/promclient/label.go:133:12: assignment mismatch: 2 variables but parser.Walk returns 1 values
#13 161.7 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/promclient/label.go:133:26: too many arguments in call to parser.Walk
#13 161.7 	have (context.Context, *LabelFilterVisitor, *parser.EvalStmt, parser.Expr, nil, nil)
#13 161.7 	want (parser.Visitor, parser.Node, []parser.Node)
#13 161.7 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/promclient/label.go:164:13: assignment mismatch: 2 variables but parser.Walk returns 1 values
#13 161.7 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/promclient/label.go:164:27: too many arguments in call to parser.Walk
#13 161.7 	have (context.Context, *LabelFilterVisitor, *parser.EvalStmt, parser.Expr, nil, nil)
#13 161.7 	want (parser.Visitor, parser.Node, []parser.Node)
#13 162.3 # github.com/jacksontj/promxy/pkg/remote
#13 162.3 go/pkg/mod/github.com/jacksontj/promxy@v0.0.75/pkg/remote/read.go:46:10: cannot use &querier{...} (type *querier) as type storage.Querier in return argument:
#13 162.3 	*querier does not implement storage.Querier (wrong type for LabelNames method)
#13 162.3 		have LabelNames(...*labels.Matcher) ([]string, storage.Warnings, error)
#13 162.3 		want LabelNames() ([]string, storage.Warnings, error)
```

--- a/README.md
+++ b/README.md
@@ -25,10 +25,11 @@ can have a single source and you can have globally aggregated promql queries.
 ## Quickstart
 Release binaries are available on the [releases](https://github.com/jacksontj/promxy/releases) page.
 
-If you are interested in hacking on promxy (or just running your own build), you can install via `go get`:
+If you are interested in hacking on promxy (or just running your own build), you can clone and build`:
 
 ```
-go get -u github.com/jacksontj/promxy/cmd/promxy
+git clone git@github.com:jacksontj/promxy.git
+cd promxy/cmd/promxy && go build
 ```
 
 An example configuration file is available in the [repo](https://github.com/jacksontj/promxy/blob/master/cmd/promxy/config.yaml).

